proposals/NNNN-linalg-matrix.md

```diff
@@ -0,0 +1,110 @@
+<!-- {% raw %} -->
+
+# Linear Algebra Matrix
+
+* Proposal: [NNNN](NNNN-linalg-matrix.md)
+* Author(s): [Chris Bieneman](https://github.com/llvm-beanz)
+* Sponsor: TBD
+* Status: **Under Consideration**
+
+## Introduction
+
+GPUs are exceptional parallel data processors, but increasingly it is becoming
+important to model operations with cross-thread data dependencies. In HLSL and
+Direct3D these operations have been called Wave operations, or in Vulkan
+Subgroup operations. Related terms like Quad or derivatives have similar
+meaning in different scoping contexts. Vulkan has also recently introduced the
+term "cooperative" when talking about operations that require participation from
+multiple threads, these can be viewed much like derivative operations but across
+the full SIMD unit instead of a subset of threads.
+
+All of these terms refer to the way the underlying instructions execute, not
+necessarily what they do. One big part of this proposal is to take 5 steps back
+and talk about what they do: linear algebra.
+
+## Motivation

```

Comment:
Worth calling out relationship to https://github.com/microsoft/hlsl-specs/blob/main/proposals/0031-hlsl-vector-matrix-operations.md?